src/app/Scenes/Inbox/Components/Conversations/Composer.tsx

```diff
@@ -27,7 +28,7 @@ const Container = styled.View`
   border-bottom-color: ${themeGet("colors.black10")};
   border-bottom-width: 1px;
   padding: 10px;
-  background-color: ${(p: ContainerProps) => (p.active ? "white" : themeGet("colors.black5"))};
+  background-color: ${(p: ContainerProps) => (p.active ? "white100" : themeGet("colors.black5"))};
```

Comment:
Suggestion: We should wrap this with `themeGet` for it to work

```suggestion
  background-color: ${(p: ContainerProps) => (p.active ? themeGet("white100") : themeGet("colors.black5"))};
```